app/services/telegram/incoming_message_service.rb

```diff
@@ -100,6 +100,8 @@ def file_content_type
   def attach_files
     return unless file
 
+    return if inbox.channel.get_telegram_file_path(file[:file_id]).blank?
```

Comment:
lets assign the value to to a variable 

```
file_download_path = inbox.channel.get_telegram_file_path(file[:file_id])
return if file_download_path.blank?
```

let's also add a spec for the same case .. stubing the API call to get download path to return error